src/lsdbus/proxy.lua

```diff
@@ -62,7 +63,7 @@ function proxy:call(m, ...)
    return self:xcall(self._intf.name, m, its, ...)
 end
 
-function proxy:__call(m, ...) return self:call(m, ...) end
+function mt:__call(m, ...) return self:call(m, ...) end
```

Comment:
I agree with this change in general, but I'd prefer to keep `proxy` as the metatable, since it allows to easily add or override methods in applications.